hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -73,25 +73,25 @@ def parse_fpv_summary(str_buffer):
                         ("covered", r"^\[\d+\].*covered.*"),
                         ("undetermined", r"^\[\d+\].*undetermined.*"),
                         ("unreachable", r"^\[\d+\].*unreachable.*")]
-    fpv_summary = extract_messages_count(str_buffer, message_patterns)
+    f_summary = extract_messages_count(str_buffer, message_patterns)
 
-    fpv_summary["pass_rate"] = format_percentage(fpv_summary["proven"],
-                                                 fpv_summary["cex"] + fpv_summary["undetermined"])
-    fpv_summary["cov_rate"] = format_percentage(fpv_summary["covered"], fpv_summary["unreachable"])
+    f_summary["pass_rate"] = format_percentage(f_summary["proven"],
+                                               f_summary["cex"] + f_summary["undetermined"])
+    f_summary["cov_rate"] = format_percentage(f_summary["covered"], f_summary["unreachable"])
 
-    return fpv_summary
+    return f_summary
 
 
 def get_results(logpath):
-    '''Parse FPV log file and extract info to a dictionary'''
+    '''Parse Formal log file and extract info to a dictionary'''
```

Comment:
Real nit, but I think this doesn't need its capital. There are quite a few more examples below: maybe it would be worth trying to re-word some of them?

Reply:
I see, that makes sense. Thanks Rupert for pointing that out.